lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -880,7 +898,54 @@ def test_dataset_sort(self, server_config, query_as, sort, results):
         for i, d in enumerate(all):
             odd = i & 1
             Metadata.setvalue(d, "global.test.sequence", i)
+            Metadata.setvalue(d, "global.test.mcguffin", 100 - i)
             Metadata.setvalue(d, "user.test.odd", odd, user=test)
         query = {"sort": sort, "metadata": ["dataset.uploaded"]}
```

Comment:
Is `global.test.mcguffin` used in any of the scenarios?  (FWIW, I don't think it produces a result which is different from any of the other scenarios.  🦐)

Reply:
Ah, I see...`global.test.mcguffin` is implicitly of referenced via the reference to `global.test`...very good.